modules/troubleshooting-network-observability-must-gather.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+:_content-type: PROCEDURE
```

Comment:
The guidelines only say this blank line is actually required for assemblies (before the toc), but all the modules examples in there also have a blank here. Otherwise this looks perfect to me :nerd_face: 

```suggestion
// * networking/network_observability/troubleshooting-network-observability.adoc

:_content-type: PROCEDURE
```